### DISCUSSION TITLE
Weather Observation Station 11

### DISCUSSION THREAD

**Discussion by mah-shamim** [hafijul233/HackerRank#20] [Q&A] [answered]
Query the list of **CITY** names from **STATION** that either do not start with vowels or do not end with vowels. Your result cannot contain duplicates.

**Input Format**

The **STATION** table is described as follows:

![1449345840-5f0a551030-Station](https://user-images.githubusercontent.com/15815972/184506892-5df75bd0-c4c3-4d99-b32c-31a21dd7aaa8.jpg)

where LAT_N is the northern latitude and LONG_W is the western longitude.

**Comment by mah-shamim** [comment, marked as answer]
/*
    Enter your query here and follow these instructions:
    1. Please append a semicolon ";" at the end of the query and enter your query in a single line to avoid error.
    2. The AS keyword causes errors, so follow this convention: "Select t.Field From table1 t" instead of "select t.Field From table1 AS t"
    3. Type your code immediately after comment. Don't leave any blank line.
*/

SELECT DISTINCT CITY FROM STATION WHERE LOWER(LEFT(CITY,1)) NOT IN ('a', 'e', 'i', 'o', 'u') OR LOWER(RIGHT(CITY,1)) NOT IN ('a', 'e', 'i', 'o', 'u');

**Reply by hafijul233** [reply]
Fine answer. What databased is used

**Reply by mah-shamim** [reply]
DB2, MySQL, Oracle, MS SQL Server
